src/main/resources/com/cloudbees/hudson/plugins/folder/config/AbstractFolderConfiguration/config.jelly

```diff
@@ -3,13 +3,7 @@
     <f:section title="${%Folder}">
         <f:entry title="${%Health Metrics}"
                  description="${%List of default health metrics for Folders}">
-            <f:repeatableHeteroProperty field="healthMetrics" oneEach="true" honorOrder="true" hasHeader="true">
-                <f:entry title="">
-                    <div align="right">
-                        <f:repeatableDeleteButton/>
-                    </div>
-                </f:entry>
```

Comment:
Huh, when did this become unnecessary?

Reply:
I went back 18 years and it was there then: https://github.com/jenkinsci/jenkins/blob/dddb4598df25527220a1e0f46d1a0d25217dd4c8/core/src/main/resources/lib/form/hetero-list.jelly#L95